engine/build.gradle

```diff
@@ -10,6 +10,13 @@ dependencies {
     compile "org.terasology:gestalt-module:6.0.0"
     compile "org.terasology:gestalt-asset-core:6.0.0"
     compile "org.terasology:gestalt-entity-system:6.0.0"
+    compile(group: "com.github.java-json-tools", name: "json-schema-validator", version: "2.2.10");
```

Comment:
Please remove what is not needed (Also, `org.json` should come by default with everit schema, should not need to depend on it specifically, but check it for yourself